<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ArrayConditionLab">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose your search method</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LinearSearchBut">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Linear Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BinarySearchBut">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Dataset Size &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ArraySizeEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateBut">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Dataset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MergeSortBut">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Using Merge Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="STLSortBut">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Using STL Sort</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FindLineEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="FindBut">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Find it!</string>
    </property>
   </widget>
   <widget class="QLabel" name="InfoLab">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorsLab">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>299</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ArrayLab">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
